packages/frontend-api/src/Component/CustomerUploadedFile/CustomerUploadedFilesBatchLoader.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Component\CustomerUploadedFile;
+
+use GraphQL\Executor\Promise\Promise;
+use GraphQL\Executor\Promise\PromiseAdapter;
+use Shopsys\FrameworkBundle\Component\CustomerUploadedFile\Config\CustomerUploadedFileTypeConfig;
+use Shopsys\FrameworkBundle\Component\CustomerUploadedFile\CustomerUploadedFile;
+use Shopsys\FrameworkBundle\Component\CustomerUploadedFile\CustomerUploadedFileFacade;
+use Shopsys\FrameworkBundle\Component\CustomerUploadedFile\Exception\CustomerFileNotFoundException;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Utils\Utils;
+
+class CustomerUploadedFilesBatchLoader
```

Comment:
I don't think so. It seems appropriate in this context since files are associated with each order item. The batch loader reduces the number of queries and simplifies usage, especially since customer files don't have a direct relationship.

Reply:
that is true, ok :+1: